code/modules/reagents/reagent_containers/spray.dm

```diff
@@ -22,6 +22,10 @@
 
 
 /obj/item/weapon/reagent_containers/spray/afterattack(atom/A as mob|obj, mob/user)
+	// Make it so the bioterror spray doesn't spray yourself when you click your inventory items
+	if (istype(src, /obj/item/weapon/reagent_containers/spray/chemsprayer) && A.loc == usr)
+		return

```

Comment:
instead of this, find `obj/item/weapon/reagent_containers/spray/chemsprayer/afterattack()` (or define it if it's not already there)
It should look like this:
```DM
obj/item/weapon/reagent_containers/spray/chemsprayer/afterattack(atom/movable/A, mob/user)
	if(A.loc == user)
		return
//any code before ..() already defined
	..()
//any code after ..() already defined
```